### PR TITLE
Updates to the email sent to parents when they attach their email to their child's account

### PR DESCRIPTION
**Pull request by bethanyaconnor**
[spec](https://docs.google.com/document/d/1iOfV0Opkg9Jqw3LEmKtmq4KCWbmeCnTkSYvf-_BNYIg/edit#heading=h.hn5x7lapdeyk)

Screenshot 2020-04-14 at 9 17 13 AM

## Testing story

http://localhost-studio.code.org:3000/rails/mailers/parent_mailer/parent_email_added_to_student_account_preview

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by dju90** [comment]
Other than the one copy change I requested, lgtm!

**Comment by bethanyaconnor** [comment]
Screenshot 2020-04-14 at 1 47 24 PM

File: dashboard/app/views/parent_mailer/parent_email_added_to_student_account.haml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 %p
-  Welcome to Code.org! You have successfully linked your email to your child’s account.
-  We're thrilled that you have taken the next step in supporting #{@student.name} in their learning and exploration of
-  computer science. If you are not #{@student.name}’s parent, please reply to this email or contact support@code.org.
+  Welcome to Code.org! You have successfully linked your email to the account of your child,
+  %strong>= " #{@student.name}"
```
Comment (daynew) [inline dashboard/app/views/parent_mailer/parent_email_added_to_student_account.haml:3]: is there supposed to be `>=` ?

File: dashboard/app/views/parent_mailer/parent_email_added_to_student_account.haml [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
+  \. We're thrilled that you have taken the next step in supporting your child in their learning and exploration of
+  computer science.  If you are not the parent or guardian for this account, please reply to this email or contact support@code.org.
```
Comment (daynew) [inline dashboard/app/views/parent_mailer/parent_email_added_to_student_account.haml:5]: will `support@code.org` automatically become a mailto link?

File: dashboard/app/views/parent_mailer/parent_email_added_to_student_account.haml [hunk 3]
```diff
@@ -51,3 +55,12 @@
...
   %a{href:"https://code.org/privacy"}privacy policy
   for details.
+
+%p
+  %strong
+    %em Did you receive this email in error?
+
+%p
+  If you are not the parent or guardian of
+  %strong>= " #{@student.name}"
```
Comment (daynew) [inline dashboard/app/views/parent_mailer/parent_email_added_to_student_account.haml:65]: same comment about `>=`

**Review by daynew — APPROVED** [review]
Looks good. Minor comments

I went through the doc sentence, by sentence, and it looks like they match.
I also tested the URLs to make sure they are valid.

File: dashboard/app/views/parent_mailer/parent_email_added_to_student_account.haml (hunk 1 above)
Comment (bethanyaconnor) [reply dashboard/app/views/parent_mailer/parent_email_added_to_student_account.haml:3]: Yes. It felt weird to me to bold the period but haml adds a space between lines. The `>` disables the automatic whitespace for this line (which is also why there's a space before the student name). I'm open to suggestions on the best way to do this!

File: dashboard/app/views/parent_mailer/parent_email_added_to_student_account.haml (hunk 3 above)
Comment (bethanyaconnor) [reply dashboard/app/views/parent_mailer/parent_email_added_to_student_account.haml:65]: Same as above.

File: dashboard/app/views/parent_mailer/parent_email_added_to_student_account.haml (hunk 2 above)
Comment (bethanyaconnor) [reply dashboard/app/views/parent_mailer/parent_email_added_to_student_account.haml:5]: It does for me. I don't know if this depends on the mail client though.

**Pull request merged**